package.json

```diff
@@ -45,12 +45,7 @@
     "twing:copy": "ncp static build",
     "twing:clean": "rimraf build dist",
     "twing:dist": "npm-run-all twing:clean storybook-build-twing:* twing:copy",
-    "update-version": "lerna version --no-push --no-git-tag-version",
-    "clean": "yarn twing:clean",
```

Comment:
Having a central `clean` which runs `run-p` of both clean for php and twing will still be useful.